src/components/MapView/Layers/PointDataLayer/index.tsx

```diff
@@ -51,18 +64,23 @@ function PointDataLayer({ layer }: { layer: PointDataLayerProps }) {
       circleLayout={circleLayout}
       circlePaint={circlePaint}
       circleOnClick={(evt: any) => {
-        dispatch(
```

Comment:
In line with a converstation I had with Amit. The default `data_field` or `mesure` in this case should remain there and add `feature_info_props` as extra fields.